src/index.ts

```diff
@@ -175,6 +177,8 @@ app.use(async (req, res, next) => {
         : noRights
     } as AuthData
   } catch (e) {
+    if (!axios.isAxiosError(e))
```

Comment:
Very good catch. Would add console.error() and separate to own commit.

Reply:
This is needed for Typescript to compile and adding clean console.error() makes compiler to complain => did a little trick and added console.error().